drivers/hmis/app/graphql/types/forms/pick_list_option.rb

```diff
@@ -166,6 +172,21 @@ def self.unit_types_for_project(project, available_only: false)
         end
     end
 
+    # UNIT_TYPES pick list should only return types that are "mapped" for this project. If there are
+    # no mappings it should return all unit types, which is the default behavior.
+    def self.possible_unit_types_for_project(project)
+      unit_type_scope = Hmis::UnitType.all
+      unit_type_ids = project.unit_type_mappings.active.pluck(:unit_type_id)
+      if unit_type_ids.any?
+        unit_type_ids += project.units.distinct.pluck(:unit_type_id) # include unit types for existing units
+        unit_type_scope = unit_type_scope.where(id: unit_type_ids)
+      end
+
+      unit_type_scope
+        .order(:description, :id)
+        .map(&:to_pick_list_option)
+    end
```

Comment:
This is fine but it should replace the `unit_types_for_project` function and the `UNIT_TYPES` pick list. There is no need to have both.

Reply:
oh wait, nevermind.

Reply:
It is redundant with `UNIT_TYPES` though, which has description `Unit types. If project is specified, limited to unit types in the project.`.

Can we keep that behavior, since we are already using the UNIT_TYPES pick list when creating new units.

We should be able to remove the `POSSIBLE_PROJECT_UNIT_TYPES` pick list and just replace the last line below:

```
      when 'UNIT_TYPES'
        # If no project was specified, return all unit types
        all_unit_types = Hmis::UnitType.order(:description, :id)
        return all_unit_types.map(&:to_pick_list_option) unless relation_id.present?
        return [] unless project.present? # relation id specified but project not found
        possible_unit_types_for_project(project) # <--- this is different
```